plugins/org.python.pydev.shared_core/tests/org/python/pydev/shared_core/actions/CommentActionTest.java

```diff
@@ -32,4 +32,19 @@ public void testLineCommentUncomment() throws Exception {
         assertEquals("a = 10", uncomment.uncommentLines("##a = 10").toString());
         assertEquals(" a = 10", uncomment.uncommentLines("##  a = 10").toString());
     }
+
+    public void testLineCommentUncommentSpaced() throws Exception {
```

Comment:
Please test with something as:

```
def method():
    if True:
        a = 10
```

The result should be:

```
def method():
    # if True:
    #     a = 10
```

Reply:
You mean the result should be like this when the user selects only the function content, right?

Reply:
Yes.

Reply:
i.e.: all lines should be analyzed to know at which indent the `#` should be added.